smtp.yml

```diff
@@ -107,6 +125,18 @@ bind:
     required: true
     type: string
     default: ${request.instance_id}
+  - name: bounce_topic_arn
+    default: ${instance.details["bounce_topic_arn"]}
+    overwrite: true
```

Comment:
I'm guessing this is wrong. We don't want the user to be able to specify their own value here, it should _only_ accept the ARN that provisioning provided. Should this line be removed, or set to false?

Reply:
Nope, this is correct. `overwrite: true` in the `computed_inputs` section means that the computed input should override anything manually provided